pynamodb/connection/transactions.py

```diff
@@ -0,0 +1,124 @@
+from pynamodb.exceptions import GetError
+
+from pynamodb.constants import (
+    RETURN_VALUES_ON_CONDITION_FAILURE, ITEM, RETURN_VALUES, RESPONSES,
+    TRANSACTION_CONDITION_CHECK_REQUEST_PARAMETERS, TRANSACTION_DELETE_REQUEST_PARAMETERS,
+    TRANSACTION_GET_REQUEST_PARAMETERS, TRANSACTION_PUT_REQUEST_PARAMETERS, TRANSACTION_UPDATE_REQUEST_PARAMETERS
+)
+from pynamodb.models import _ModelFuture
+
+
+class Transaction(object):
+
+    _connection = None
```

Comment:
Same here -- why the class variables?

Reply:
do you mean why are they put outside of the `__init__`, or why do they exist at all?